### DISCUSSION TITLE
Add embedding system

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3060] [Feature Requests]
A way to embed from external services like YouTube, Vimeo, SoundCloud etc. Ideally, this is an extension type so you can install more scoped options. 

* How to handle API keys?
* Register based on URL Regex?

**Comment by earth2jay** [comment]
This sounds like it's in the area of my issue with Vimeo.  Currently, Vimeo allows you to customize the embed player by turning 'playbar' on or off.  But I want the 'playbar' to be OFF until the user requests it, by either hovering over or whatever... Am I dreaming?

**Reply by earth2jay** [reply]
UPDATE: I found a player called Wistia that does allow the customization I wanted.  Thank you!

**Comment by everyx** [comment]
> * How to handle API keys?

@rijkvanzanten If we regard this as an extension, then we may be able to add a extension specific configuration feature, same as many other CMS plugin.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
